### PR TITLE
Minor doc fixes

### PR DESCRIPTION
**Pull request by FaithOmbongi**

### CODE REVIEW

File: api-reference/beta/resources/identityprotection-overview.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Use the Azure AD identity protection APIs"
+description: "Use Microsoft Graph to query the Identity Protection APIs to receive information about risk detected by Azure AD Identity Protection."
```
Comment (Lauragra) [inline api-reference/beta/resources/identityprotection-overview.md:3]: Please make sure that we're consistently lower casing identity protection. Also a minor point, APIs is plural in the title but singular in the H1.

File: api-reference/beta/resources/identityprotection-overview.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+* [riskyUsers](riskyuser.md) - Query Microsoft Graph for information about users that Identity Protection detected as risky. User risk represents the probability that a given identity or account is compromised. These risks are calculated offline using Microsoft’s internal and external threat intelligence sources, including security researchers, law enforcement professionals, security teams at Microsoft, and other trusted sources.
+
+* [signIn](signin.md) - Query Microsoft Graph for information about Azure AD sign-ins with specific properties related to risk state, detail, and level. A sign-in risk represents the probability that a given authentication request isn’t authorized by the identity owner. These risks can be calculated in real-time or calculated offline using Microsoft’s internal and external threat intelligence sources, including security researchers, law enforcement professionals, security teams at Microsoft, and other trusted sources.
+
+
+## What can I do with Identity Protection APIs in Microsoft Graph?
```
Comment (Lauragra) [inline api-reference/beta/resources/identityprotection-overview.md:28]: Lower case the reference to Identity Protection.

File: api-reference/v1.0/resources/identityprotection-overview.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Use the Azure AD identity protection APIs"
```
Comment (Lauragra) [inline api-reference/v1.0/resources/identityprotection-overview.md:2]: Please make the same minor fixes in this topic as well.

File: api-reference/beta/resources/identityprotection-overview.md (hunk 1 above)
Comment (FaithOmbongi) [reply api-reference/beta/resources/identityprotection-overview.md]: Resolved.

File: api-reference/v1.0/resources/identityprotection-overview.md (hunk 3 above)
Comment (FaithOmbongi) [reply api-reference/v1.0/resources/identityprotection-overview.md]: Resolved.

File: api-reference/beta/resources/identityprotection-overview.md (hunk 2 above)
Comment (FaithOmbongi) [reply api-reference/beta/resources/identityprotection-overview.md]: Resolved

**Comment by FaithOmbongi** [comment]
Hi @Lauragra, I hope I've properly addressed the feedback in the identity protection API docs. Kindly re-review. Thanks.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**